frontend/src/metabase/collections/components/CollectionHeader/CollectionInfoSidebarToggle.tsx

```diff
@@ -17,9 +16,12 @@ export const CollectionInfoSidebarToggle = ({
   const [showSidesheet, setShowSidesheet] = useState(false);
   return (
     <>
-      <CollectionHeaderButton onClick={() => setShowSidesheet((open) => !open)}>
-        <Icon name="info" />
-      </CollectionHeaderButton>
+      <ToolbarButton
+        icon="info"
+        aria-label={t`More info`}
+        tooltipLabel={t`More info`}
```

Comment:
!nit can we make `ToolbarButton` automatically set `aria-label` as `tooltipLabel` if it's not defined?

Reply:
If I ever call out a nit you can always merge if you want, there's no pressure to do it in this PR. I just noticed the duplication in this PR and components like `SharingMenu.tsx`, `AIQuestionAnalysisButton.tsx`, etc. that were doing this already.